src/odroid.md

```diff
@@ -8,14 +8,15 @@
  * Power board on|off loop
  * Odroid power cable (make sure this is plugged in to a 5V port)
  * Micro USB cable
- * USB stick (with the following code in a robot.zip from the IDE):
+ * USB stick (with the following code in a robot.zip):
```

Comment:
Suggestion: Should we link out to https://studentrobotics.org/docs/programming/getting_code_on_the_robot here?